Source/BugsnagNotifier.m

```diff
@@ -287,6 +290,21 @@ - (void) start {
 #endif
 }
 
+- (void)setupConnectivityListener {
+    NSString *hostUrl = [self apiHostUrl];
+    self.apiReachable = [BSGReachability reachabilityWithHostName:hostUrl];
+    [self.apiReachable startNotifier];
+    
+    __weak id weakSelf = self;
+    self.apiReachable.reachableBlock = ^(BSGReachability *reachability) {
+        [weakSelf flushPendingReports];
+    };
+}
+
+- (NSString *)apiHostUrl {
```

Comment:
Its a bit unconventional for a method which returns a string to be named 'url'. This could also be scoped to `setupConnectivityListener` since its only used in the single place.